File: LUMINAR/Looping/primeinbtw.py
low=int(input("enter a  lower limit: "))
upper=int(input("enter upper limit: "))


for j in range(low,upper+1):
    if (j>1):
        for i in range(2, j):
            if (j % i) == 0:  # 5%(2,,,,4), 9%3==0 True    9%2==0 False
                break
        else:
            print(j, end=" ")

File: LUMINAR/VS_code/html_progs/javascript/for_loop/prime_in_btw.js
var lower = Number(prompt("enter the lower limit: "))
var upper = Number(prompt("enter the upper limit: "))


for (i = lower; i <=upper; i++) {
    flag = 0;
    for (j = 2; j < i; j++) {
        if (i % j == 0) {
            flag = 1
            break
        }
    }
    if (flag == 0) {
        console.log(i)
    }
}
